cypress/e2e/helpers/surveyHelper.ts

```diff
@@ -82,6 +91,117 @@ export const verifyWeightScreenContent = (
   );
 };
 
+export const verifyOnboardingExperienceOptions = (
```

Comment:
Just to keep same pattern as the others
```suggestion
export const verifyOnboardingExperienceScreen = (
```

Reply:
It's true, it gets better.